### ISSUE TITLE
Component for Displaying Service Status

### ISSUE THREAD

**Issue by dtaboada** [adamdriscoll/poshprotools#238]
Just curious if there was a component to leverage service status, I understand currently we can use New-UDTable and invoke get service to display results, but I'm interested in lighting up certain areas based on status (stopped, started etc...) of critical services.

**Comment by adamdriscoll** [comment]
So you are looking for a table that changes colors when services are different statuses? Or is it more than that?

**Comment by kort3x** [comment]
Conditional row highliting would indeed be awesome.

Just like alternating row colors and highliting on hover.

**Comment by dtaboada** [comment]
Yes, one that will change based on service statuses, maybe an item/icon in a column that we can change dynamically, not sure how aesthetically that would look like but just a thought ;) Also would be nice to have an event driven menu on that item, that can perform onclick action.

**Comment by dtaboada** [comment]
Actually it would be neat if we can pass the component a bool value, that way usability can be expanded to all sorts of scenarios, for instance if I have a check list of items for a particular application, and to further add we can then roll these status's up to bigger peice that can show the "overall" or "highlevel" status?

**Comment by adamdriscoll** [comment]
Sure. It sounds like a great idea for a custom component. Once 1.4.0 ships I'll put something together.
